app/Http/Controllers/CoverageController.php

```diff
@@ -71,11 +72,9 @@ public function manageCoverage(): View|RedirectResponse
         }
 
         // If the projectid is not set and there is only one project we go directly to the page
-        if (!isset($projectid)) {
-            $projectids = $Project->GetIds();
-            if (count($projectids) == 1) {
-                $projectid = $projectids[0];
-            }
+        // TODO: (williamjallen) Should this be one project in all of CDash, or one project we can see?
+        if (!isset($projectid) && EloquentProject::count() === 1) {
+            $projectid = EloquentProject::all()->firstOrFail();
```

Comment:
I get a 500 error at line 79 thanks to this line reporting that you cannot convert `App\Models\Project` to `int`.   I only have a single project on the system so I think this needs another look: 

![image](https://github.com/Kitware/CDash/assets/542106/3a144cee-cfe9-4777-8135-eec6eb3cebbe)

Reply:
Good catch!  Fixed.